packages/page-staking/src/Nomination/index.tsx

```diff
@@ -0,0 +1,380 @@
+  }
+
+  function resetControllerInfo(accountId: string | null) {
+    setControllerAlreadyBonded(false);
+    setControllerAccountId(accountId);
+  }
+
+  /**
+   * double fees for case if fees will be changed on a small count of funds
+   * @todo - compare with account balance and throw error if more
+   */
+  function setAmountToTransfer() {
+    const minAmount = new BN(0);
+    setTransferableAmount(
+      minAmount
+        .iadd(wholeFees)
+        .iadd(wholeFees)
+        .isub(controllerBalance || new BN(0))
+    );
+  }
+
+  function calculateMaxPreFilledBalance() {
+    if (accountBalance && wholeFees && !amountToBond) {
+      // double wholeFees
+      setAmountToBond(accountBalance.isub(wholeFees).isub(wholeFees).isub(existentialDeposit));
+    }
+    return 0;
+  }
+
+  function isBalanceEnough() {
+    return (accountBalance
+      && controllerBalance
+      && existentialDeposit
+      && wholeFees
+      && accountBalance.cmp(existentialDeposit) === 1
+      && controllerBalance.cmp(wholeFees) === 1)
+  }
+
+  const _onUpdateControllerState = useCallback(
+    (controllerAlreadyBonded: boolean): void => {
+      setControllerAlreadyBonded(controllerAlreadyBonded);
+    },
+    [controllerAlreadyBonded]
+  );
+
+  const _onUpdateNominatedState = useCallback(
+    (isNominated) => {
+      setIsNominated(isNominated);
+    },
+    [isNominated]
+  );
+
+  /**
+   * Set validators list.
+   * If filtered validators
+   */
+  useEffect(() => {
+    if (validators && validators.length) {
+      setSelectedValidators(
+        validators.map((validator): string => validator.key).slice(0, 16)
+      );
+    } else {
+      stakingOverview && setSelectedValidators(
+        stakingOverview.validators.map((acc): string => acc.toString()).slice(0, 16)
+      );
+    }
+  }, [validators, stakingOverview]);
+
+  useEffect(() => {
+    if (!wholeFees) {
+      return;
+    }
+    setAmountToTransfer();
+    calculateMaxPreFilledBalance();
+  }, [accountBalance, controllerBalance, wholeFees]);
+
+  useEffect(() => {
+    setStepsStateAction();
+  }, [controllerAlreadyBonded, isNominated]);
+
+  /**
+   * Since we already have stashes just open the 4th screen - nomination
+   * and mark all steps as completed
+   */
+  /* useEffect(() => {
+    setCurrentStep(steps[3]);
+    setAlreadyHaveStashes(true);
+    setStepsState(['completed', 'completed', 'completed', 'completed']);
+  }, [ownStashes]); */
+
+  return (
+    <main className={`${className} ${!isVisible ? 'staking--hidden' : ''} simple-nominatio`}>
+      <TabsHeader
+        setCurrentStep={setCurrentStep}
+        stepsState={stepsState}
+        steps={steps}
+        currentStep={currentStep}
+      />
+      <div className="ui attached segment">
+        {/* {feesLoading && (
+          <Spinner />
+        )} */}
+        {currentStep === steps[0] &&
+        <>
+            <br />
+            <h3>{t('Select your account that holds funds')}:</h3>
+            <br />
```

Comment:
Rather adjust the styl;e with consistent padding/margins, instead of using `<br />` to adjust the layouts.